build.yaml

```diff
@@ -181,7 +182,7 @@ build:
       pip install --upgrade pip
       pip install -U setuptools
 
-      pip install git+ssh://git@github.com/riptano/ccm-private.git
+      pip install $HOME/ccm
```

Comment:
Is ccm already cloned here? What's this do?

Reply:
Yes. I wasn't aware but ccm-private is cloned when building driver images.